### PR TITLE
create object and fix ui

### PR DESCRIPTION
**Pull request by VuManhHung97**
Create model : Collection, Photo , Location, ConvertPhoto, ProfileImage,UrlImage, User
Fix UI : Activity to Fragment

https://edu-redmine.sun-asterisk.vn/issues/14607

### CODE REVIEW

**Comment by VuManhHung97** [comment]
ready

File: app/src/main/res/layout/fragment_home.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_home.xml]:
```suggestion
<android.support.constraint.ConstraintLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
```

File: app/src/main/java/com/sun_asterisk/myeditor03/utils/FragmentHelper.kt [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package com.sun_asterisk.myeditor03.utils
+
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentTransaction
+
+object FragmentHelper {
+    fun goNextChildFragment(
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/myeditor03/utils/FragmentHelper.kt]: dùng extension

**Comment by VuManhHung97** [comment]
ready

File: app/src/main/java/com/sun_asterisk/myeditor03/utils/ActivityExtention.kt [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    if (currentFragment == null) {
+        currentFragment = fragment
+        transaction.add(containerViewId, fragment, fragment.javaClass.simpleName)
+    }
+    if (addToBackStack) {
+        transaction.addToBackStack(rootTag)
+    } else {
+        showFragment(supportFragmentManager!!, currentFragment)
+    }
+    transaction.commit()
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/myeditor03/utils/ActivityExtention.kt:18]: thử xem được không e
```
fun AppCompatActivity.addFragmentToActivity(
    @IdRes containerViewId: Int,
    fragment: Fragment,
    addToBackStack: Boolean = true,
    tag: String = fragment::class.java.simpleName
) {
    supportFragmentManager.beginTransaction()
    val currentFragment = supportFragmentManager!!.findFragmentByTag(fragment.javaClass.simpleName)
    val transaction = supportFragmentManager.beginTransaction()
    if (currentFragment != null) {
        transaction.add(containerViewId, fragment, fragment.javaClass.simpleName)
    }
    if (addToBackStack) {
        transaction.addToBackStack(tag)
    }
    transaction.show(fragment)
    transaction.commit()
}
```

Comment (VuManhHung97) [reply app/src/main/java/com/sun_asterisk/myeditor03/utils/ActivityExtention.kt:18]: vâng anh

**Comment by VuManhHung97** [comment]
ready

File: app/src/main/java/com/sun_asterisk/myeditor03/utils/ActivityExtention.kt [hunk 4]
```diff
@@ -3,36 +3,21 @@ package com.sun_asterisk.myeditor03.utils
 import android.support.annotation.IdRes
 import android.support.v4.app.Fragment
 import android.support.v4.app.FragmentManager
-import android.support.v4.app.FragmentTransaction
 import android.support.v7.app.AppCompatActivity
 
-fun AppCompatActivity.goNextChildFragment(
+fun AppCompatActivity.goToFragment(
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/myeditor03/utils/ActivityExtention.kt]:
```suggestion
fun AppCompatActivity.addFragmentToActivity(
```

**Comment by VuManhHung97** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**